src/views/pages/ManagePeople/index.vue

```diff
@@ -52,81 +53,73 @@ export default Vue.extend({
       this.searched = searchByName(this.people, this.search);
     },
   },
-  created: function created() {
-    this.searched = this.people;
+  created: async function created() {
+    const self = this;
+
+    // get people types
+    const personCategories: any[] = [];
+    await axios.get('http://boresha.tech:9090/people/categories')
+      .then(async function (response: any) {
+        response.data.forEach(function(category: any) {
+          personCategories.push(category.name);
+        });
+      })
+      .catch(async function (error: any) {
+        console.log(error.message);
+      });
+
+    // get all people
+    personCategories.forEach(async function(category: string) {
+      // get all people of particular category
+      axios.get('http://boresha.tech:9090/people/' + category)
+        .then(async function (response: any) {
+          // construct each person
+          response.data.forEach(function (person: any) {
+            // construct full name
```

Comment:
Try looking into string templating in es6 would make this far easier and cleaner